### PR TITLE
Add vjsamuel to 2023 candidacy list

### PR DESCRIPTION
**Pull request by vjsamuel**

### CODE REVIEW

File: elections/2023/governance-committee-candidates.md [hunk 1]
```diff
@@ -141,4 +141,23 @@ and welcoming nature of the OTel community, as well as their openness and
...
+- GitHub: [vjsamuel](ttps://github.com/vjsamuel)
+
+Vijay Samuel is a Principal MTS, Architect at eBay and helps lead its internal Observability platform. He has been involved
+in the OpenTelemetry project for the last couple of years and has been a strong advocate for the project both within eBay
+and externally. Vijay has given talks at [KubeCon](https://www.youtube.com/watch?v=tZJd6W-CIcU), 
+done [panel discussions](https://www.youtube.com/watch?v=cypL2ZWUElE&t=9278s) and written 
+technical [blogs](https://opentelemetry.io/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/) on the advantages of
+adopting OpenTelemetry within large organizations such as eBay. Vijay is also the co-chair of TAG Observability's Query
+Language Standardization working group. Vijay is passionate about Open Source and has contributed to various projects since the age
+of 19.  He has a wealth of knowledge not only in being an adopter of Open Telemetry but also in building out massive scale
```
Comment (jpkrohling) [inline elections/2023/governance-committee-candidates.md:159]: s/Open Telemetry/OpenTelemetry/

**Review by jpkrohling — APPROVED** [review]
LGTM!

**Comment by svrnm** [comment]
I support

**Review by codeboten — APPROVED** [review]
🎉

**Pull request merged**